### PR TITLE
feat: create collection service

### PR DESCRIPTION
**Pull request by MAudelGisaia**

### CODE REVIEW

**Review by sebbousquet — APPROVED** [review]

File: projects/arlas-components/src/lib/pipes/get-collection-display-name/get-collection-display-name.pipe.ts
Comment (QuCMGisaia) [inline projects/arlas-components/src/lib/pipes/get-collection-display-name/get-collection-display-name.pipe.ts:1]: Missing licence

File: projects/arlas-components/src/lib/pipes/get-collection-display-name/get-collection-display-name.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export class GetCollectionDisplayNamePipe implements PipeTransform {
+
+  public constructor(private collectionService: CollectionService) {
+  }
+
+  public transform(value: string, ...args: unknown[]): string {
+    if(!value) {
+      return  '';
+    }
+    return  this.collectionService.getDisplayName(value);
```
Comment (QuCMGisaia) [inline projects/arlas-components/src/lib/pipes/get-collection-display-name/get-collection-display-name.pipe.ts:17]: extra space after return

File: projects/arlas-components/src/lib/pipes/get-collection-unit/get-collection-unit.pipe.ts
Comment (QuCMGisaia) [inline projects/arlas-components/src/lib/pipes/get-collection-unit/get-collection-unit.pipe.ts:1]: Missing licence

File: projects/arlas-components/src/lib/services/collection.service.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+export abstract class BaseCollectionService {
+  protected  abstract _initUnits();
+  protected  abstract _initDisplayNames();
+  public abstract  getDisplayName(collectionName: string): any;
```
Comment (QuCMGisaia) [inline projects/arlas-components/src/lib/services/collection.service.ts:7]: Shouldn't the returned types be string ?

**Review by QuCMGisaia — APPROVED** [review]

**Review by MohamedHamouGisaia — APPROVED** [review]

**Comment by MohamedHamouGisaia** [comment]
LGTM, you can merge !

**Pull request merged**